test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/concurrent/atomic/LongAdderTest.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.util.concurrent.atomic
+
+import org.junit.Test
+import org.junit.Assert._
+
+class LongAdderTest {
+
+  @Test def longAdderIncrementTest(): Unit = {
+    val value = new java.util.concurrent.atomic.LongAdder
+    assertEquals(0L, value.sum())
+    value.increment()
+    assertEquals(1L, value.sum())
+    assertEquals(1L, value.longValue())
+    assertEquals(1, value.intValue())
```

Comment:
Consider testing `longValue`, `intValue`, `floatValue` and `doubleValue` in a dedicated test (e.g., using `add()` to directly set a value to test against). They are not part of the core, "intended" API for `LongAdder`, and so having them in all the tests of the other methods is distracting.

Reply:
Thanks @sjrd, I have added more isolated tests.